src/test/resources/com/puppycrawl/tools/checkstyle/checks/header/regexpheader/InputRegexpSmallHeader.java

```diff
@@ -0,0 +1,6 @@
+package com.puppycrawl.tools.checkstyle.checks.header.regexpheader;
+
+
+/**
+ */
+public class InputRegexpSmallHeader {}
```

Comment:
@sharang108
It must **start** with `InputRegexpHeader`, so you can append anything appropriate.